RevenueCatUI/CustomerCenter/ViewModels/CustomerCenterViewModel.swift

```diff
@@ -140,6 +144,18 @@ import RevenueCat
         self.state = .success
     }
 
+    convenience init(
+        activeSubscriptionPurchases: [PurchaseInformation],
+        activeNonSubscriptionPurchases: [PurchaseInformation],
+        configuration: CustomerCenterConfigData
+    ) {
+        self.init(actionWrapper: CustomerCenterActionWrapper(legacyActionHandler: nil))
+        self.activeSubscriptionPurchases = activeSubscriptionPurchases
+        self.activeNonSubscriptionPurchases = activeNonSubscriptionPurchases
+        self.configuration = configuration
+        self.state = .success
+    }
+
```

Comment:
Total nit: if this is only used in tests, perhaps it should be declared in the tests target?

Reply:
Ok I kind of like that. I think there's a bunch of initializers that can be moved. I'll apply that in another PR to keep it clean.